website/src/pages/plugins.js

```diff
@@ -124,20 +124,31 @@ const SectionTitle = styled.h2`
 
 
 function Plugins(props) {
+  useEffect(() => {
+    const script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.async = true;
+    script.src = '/js/plugin-icon.js';
+    document.head.appendChild(script);
+  }, []);
```

Comment:
Is it possible to use this method?

https://docusaurus.io/docs/next/docusaurus.config.js/#scripts

Reply:
Yes, it works. But it will load this JS globally and maybe reduce the loading speed of other page.